docs/generate-iam-policies.jq

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/jq -Mfr
```

Comment:
Prefer long arguments for committed files for readability

Reply:
Not actually using this method of invoking as it didn't work .. I'll just remove it